mutt/file.h

```diff
@@ -39,19 +40,6 @@ typedef uint8_t ReadLineFlags;             ///< Flags for mutt_file_read_line(),
 #define MUTT_RL_CONT      (1 << 0) ///< \-continuation
 #define MUTT_RL_EOL       (1 << 1) ///< don't strip `\n` / `\r\n`
 
-#ifdef HAVE_STRUCT_TIMESPEC
-struct timespec;
-#else
-/**
- * struct timespec - Time value with nanosecond precision
- */
-struct timespec
-{
-  time_t tv_sec; ///< Number of seconds since the epoch
-  long tv_nsec;  ///< Number of nanosecond, on top
-};
-#endif
-
```

Comment:
`HAVE_STRUCT_TIMESPEC` is defined by our `configure` script.

Reply:
Oh, so it is! I think I got this from my initial test with `-Werror` where it didn't get picked up. Dropped.